documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,304 @@
+
+The `virtual machine preferences` have replaced {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+Custom mappings of guest operating system type to virtual machine preference can be configured using `config` maps. This is in order to use custom virtual machine preferences, or to support more guest operating system types.
+
+.Full support for migration from OVA
+
+{project-short} migration from OVA is now a fully supported feature, and no longer Technical Preview.
+
+.The VM `Running` state is displayed during the migration
+
+{project-short} shows each VM's migration `Running` state during the migration. This single `Running` operation replaces two previous operations, `Create/Post VM` and Run VM`.  link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.The `must-gather` logs can now be loaded only by using the CLI
+
+The {product-short} web console can no longer download logs. With this update, you must download every `must-gather` log image by using CLI commands. for more information, see link:https://github.com/openshift/must-gather-operator[Must Gather Operator].
+
+.Migrated VMs can now be reviewed and configured
+
+The {project-short} web console now displays the name and migration status of each virtual machine (VM). The migrated VMs have the `app: <vmname>` label to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This part provides features and enhancements introduced in {project-full} 2.6:
+
+.Validation for RHEL 6 guest operating system is no available
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system. Therefore, RHEL 6 is not supported in {ocp-name} Virtualization. With this update, a validation of RHEL 6 guest operating system was added to {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Migrations from {ocp-name} versions before 4.13 are now blocked
+
+The `Export API` function in {ocp-name} Virtualization now exposes VM manifests from {ocp-name} 4.13. Therefore, source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+.Source and storage endpoints are now not selected automatically
+
+Source network and source storage are now not selected automatically when creating a {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
+
+.Automatic retrieval of CA certificates for the provider's URL in the console
+
+The ability to retrieve CA certificates, which was available in previous versions, has been restortes. The `vSphere Verify certificate` option is in the `add-provider` dialog. This option was removed in the transition to {ocp} console, and has now been added to the console. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
```

Comment:
we can also say that this functionality is also available for RHV, OpenStack and OpenShift providers now